js/main.js

```diff
@@ -8,6 +10,28 @@ search.addEventListener('click', function (event) {
   xhr.open('GET', 'https://pokeapi.co/api/v2/pokemon/' + input);
   xhr.responseType = 'json';
   xhr.addEventListener('load', function () {
+    if (xhr.status === 200) {
+      $searchAddMember.classList.remove('hidden');
+      var $imageTitle = document.createElement('h2');
+      $imageTitle.setAttribute('class', 'image-title');
+      $imageTitle.textContent = input;
+      $imageName.appendChild($imageTitle);
+      var $addButton = document.createElement('i');
+      $addButton.setAttribute('class', 'fa-solid fa-plus fa-2x add-image-icon');
+      $imageName.appendChild($addButton);
+      var $image = document.createElement('img');
+      $image.setAttribute('class', 'search-img');
+      $image.setAttribute('src', xhr.response.sprites.front_default);
+      $imageName.appendChild($image);
+      var $addButtonListener = document.querySelector('.add-image-icon');
+      $addButtonListener.addEventListener('click', function (event) {
```

Comment:
This addEventListener shouldn't be within the callback function for the 'load' event. 

You should be able to locate this outside of the function. Because of scoping reasons you may not be able to do that. 

If that is the case you would need to do DOM event delegation. You would have an addEventListener on wherever it is being appended to your HTML.